.github/ISSUE_TEMPLATE/30-impl-proposal.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Implementation proposal
```

Comment:
I thought a bit on this. Since implementation proposal needs to solve at least one use case, why don't we just add implemenation ideas on that use case and link to the related use case?